### PR TITLE
Compact output by default when listing objects

### PR DESCRIPTION
**Pull request by kgilpin**
**What is this?**

The new default behavior when listing objects is to display the object ids only.

Full object information can be displayed using `--inspect`.

**How to verify?**

You can create a sandbox using `./dev/start.sh`. Then load a policy, and list objects using `conjur list`. Note that the output is non-verbose.

The API key of the `admin` user in the `cucumber` account can be obtained using the Docker logs of the possum container.

### CODE REVIEW

File: dev/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+pg:
+  image: postgres:9.3
+  
+possum:
+  image: possum
+  command: server -a cucumber
+  environment:
+    DATABASE_URL: postgres://postgres@pg/postgres
+    POSSUM_DATA_KEY:
+  links:
```
Comment (dustinmm80) [inline dev/docker-compose.yml]: i'm pretty sure you don't need links here anymore, now that `docker-compose` services all use the same network by default. You already named the service `pg` up above so it should just work if you:
1. add `version: '2'` at the top of the file
2. nest the different containers under a new `services:` block

File: dev/start.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -ex
+
+export COMPOSE_PROJECT_NAME=clirubydev
+
+docker-compose build
+
+if [ ! -f data_key ]; then
+	echo "Generating data key"
+	docker-compose run --no-deps --rm --entrypoint possum possum data-key generate > data_key
```
Comment (dustinmm80) [inline dev/start.sh]: what is possum's original entrypoint? has that changed recently?

File: dev/stop.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -ex
+
+export COMPOSE_PROJECT_NAME=clirubydev
```
Comment (dustinmm80) [inline dev/stop.sh]: You can replace all 3 of these lines with

```
COMPOSE_PROJECT_NAME=clirubydev docker-compose down -v
```

Comment (kgilpin) [inline dev/stop.sh]: Ok, but how is that better?

**Comment by kgilpin** [comment]
@dustinmm80 This is ready to be merged IMO.

Just confirm https://jenkins.conjur.net/job/cli-ruby-pipeline/job/feature%252Fless-verbose/3/ first.

File: dev/stop.sh (hunk 3 above)
Comment (kgilpin) [inline dev/stop.sh]: OK I see, I didn't really understand the comment before.

**Pull request merged**